test/lsp/protocol_test_corpus.cc

```diff
@@ -221,7 +221,7 @@ TEST_P(ProtocolTest, DoesNotMergeFileChangesAcrossNonDelayableRequests) {
     requests.push_back(openFile("foo.rb", "# typed: true\n\nclass Opus::CIBot::Tasks::Foo\n  extend T::Sig\n\n  sig "
                                           "{returns(Integer)}\n  def bar\n    false\n  end\nend\n"));
     // Should block ^ and V from merging.
-    requests.push_back(workspaceSymbol("foo.rb"));
```

Comment:
workspaceSymbol is now delayable; it doesn't rely on the latest edits to run. I had to switch to hover to fix the test.